src/ports/postgres/modules/graph/test/pagerank.sql_in

```diff
@@ -84,7 +89,8 @@ SELECT pagerank(
              NULL,
              NULL,
              NULL,
-             'user_id');
+             'user_id',
+             NULL);
 
 -- View the PageRank of all vertices, sorted by their scores.
 SELECT assert(relative_error(SUM(pagerank), 1) < 0.00001,
```

Comment:
We may need at least two test cases with nodes_of_interest not null and with/without grouping.